concourse/scripts/common.bash

```diff
@@ -146,7 +146,16 @@ function configure() {
     # The full set of configure options which were used for building the
     # tree must be used here as well since the toplevel Makefile depends
     # on these options for deciding what to test. Since we don't ship
-    ./configure --prefix=/usr/local/greenplum-db-devel --disable-orca --enable-gpcloud --enable-orafce --enable-tap-tests --with-gssapi --with-libxml --with-openssl --with-perl --with-python --with-uuid=e2fs --with-llvm --with-zstd PYTHON=python3.11 PKG_CONFIG_PATH="${GPHOME}/lib/pkgconfig" ${CONFIGURE_FLAGS}
+
+	# temp to run tests on ci
+	export CC=clang
+	export CXX=clang++
+	if [ -z "${CC}" ] || [ -z "${CXX}" ]; then
```

Comment:
Why are these variables checked if they have been set in the previous line? Do we need to remove the export commands to use variables from ARG-s from arenadata/Dockerfile?

Reply:
CC and CXX were hardcoded because there's no configurable pipeline at the moment where we could choose a compiler, so the check is required

Reply:
If you want to remove exports, I suggest to replace `else` with `fi` because of `exit 1` and don't change the line with `./configure`